AppHost/Program.cs

```diff
@@ -4,13 +4,12 @@
 
 var sqlPassword = "YourSecretPassword01!"; 
 
-var sqlServer = builder.AddSqlServerContainer("localhost", sqlPassword, 1433)
+var sqlServer = builder.AddSqlServerContainer("127.0.0.1", sqlPassword, 1433)

```

Comment:
This isn't the address it's the name of the resource.

```suggestion
var sqlServer = builder.AddSqlServerContainer("default", sqlPassword, 1433)

```